codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -335,8 +334,8 @@ private static void writeChildNode(FlowState state) {
     /**
      * The outbound flow control state for a single stream.
      */
-    final class FlowState {
-        private final Deque<Frame> pendingWriteQueue;
+    private class FlowState {
```

Comment:
I removed it because, what's the point of a private final class? Nobody can extend it anyway.